### DISCUSSION TITLE
Optional Notifications

### DISCUSSION THREAD

**Discussion by NicosNicolaou16** [community/community#78911] [Mobile] [answered]
It is a good idea to have optional notifications when someone ⭐ a repository, fork a repository. 😀🙏

**Comment by stevepopovich** [comment]
Hi @NicosNicolaou16 am I correct in understanding you want a notification when someone stars or forks one of your repositories? Would you want a push notification or inbox notification?

**Reply by NicosNicolaou16** [reply]
Yes for the repository! Both, if it is possible, but with the choice to disable/enable the notifications because for someone it will be  annoying.

**Reply by stevepopovich** [reply, marked as answer]
For sure, if we ever implement this, we would add a setting in the profile to toggle them. Thanks, I'll pass this onto the team for consideration

**Reply by NicosNicolaou16** [reply]
Thank you very much! 😀
